partcad/src/partcad/part.py

```
@@ -13,33 +13,35 @@
 from . import logging as pc_logging
+from .user_config import UserConfig
 
 
 class Part(ShapeWithAi):
     path: typing.Optional[str] = None
     url: typing.Optional[str] = None
     count: int = None
 
-    def __init__(self, project_name: str, config: dict = {}, shape=None):
-        super().__init__(project_name, config)
+    def __init__(self, project_name: str, config: dict = {}, shape=None, user_config: UserConfig=None):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)